### ISSUE TITLE
Setting useLevel to 'silent' in config results in TypeError

### ISSUE THREAD

**Issue by louiscenter** [pinojs/express-pino-logger#9]
If you set the `useLevel` property in `pino`'s config object to `'silent'`, it results in the following:

```
TypeError: log[useLevel] is not a function
```

Is this because either `express-pino-logger` or `pino-http` are using an older version of `pino`? I'm happy to make a PR for this but I'm not sure where to start with this particular issue. Any pointers are appreciated.

**Comment by davidmarkclements** [comment]
Hey @louiscenter can you show a small code snippet just so we know exactly whats happening

First check - could you try upgrading the pino that pino-http is using

Cc @jsumners

**Comment by jsumners** [comment]
Yeah, I'd like to see how the instantiation is being done. What is this `useLevel` property?

**Comment by davidmarkclements** [comment]
It's a pino-http option 

This is the line https://github.com/pinojs/pino-http/blob/master/logger.js#L43

Upgrading pino won't help 

So what's happened is we introduced silent, but the side effect is pino-http expects the specified level to be a function 

We could special case it (and probably would have to special case it in hapi-pino as well) but it does mean any loggers that sit on top of pino will have to know that silent is a special case 

Think I'd prefer to patch pino with silent as noop - what do you think @mcollina @jsumners

**Comment by jsumners** [comment]
I think just patch `pino-http` to support the special case. `pino` has already documented that to enable "silent" mode you should set `enabled: false`.

As for `hapi-pino`, I always just pass in my own instance of `pino` and use `req.logger`. I haven't had any issues.

**Comment by louiscenter** [comment]
Hey! Thanks for hopping on this so promptly.

@davidmarkclements @jsumners - if it's of any use still, here is a small code example of what I was doing:

```
var app = require('express')()

var pino = require('express-pino-logger')({
  useLevel: 'silent'
})

app.use(pino)

app.get('/', function (req, res) {
  res.send('Hello world')
})

app.listen(3000, function () {
  console.log('Listening on port 3000!')
})
```

After I noticed `express-pino-logger` was simply using `pino-http`, I checked its documentation. I saw that `useLevel` was a property you could include in an option object, and I noticed in the `pino` documentation that you could set the `level` property to `silent`, but I wrongly assumed this would work out of the box for `pino-http`.

Is it useful if I make a ticket in the `pino-http` repo instead?

**Comment by davidmarkclements** [comment]
@louiscenter hold off a sec on that - depending on convo with @jsumners and @mcollina it may be better to deal with pino core (to avoid/solve similar scenarios)

**Comment by jsumners** [comment]
I created PR https://github.com/pinojs/pino/pull/199 to implement the proposed solution. I don't have a strong feeling either way. But @mcollina might; I think he is a proponent of `silent` being a private method.

**Comment by davidmarkclements** [comment]
I've released pino 4.0.2 and pino-http 2.4.2 - express-pino-logger and the rest will pull the patch version 

thanks @jsumners and thanks @louiscenter for reporting
